app/src/main/res/layout/fragment_assets.xml

```diff
@@ -10,7 +10,152 @@
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:orientation="vertical">
+        android:orientation="vertical"
+        android:background="@color/surface_01">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/total_balance"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <TextView
+                android:id="@+id/total_balance_label"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="16dp"
+                android:layout_marginTop="16dp"
+                android:layout_marginEnd="16dp"
+                android:text="@string/coins_total_balance"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <TextView
+                android:id="@+id/total_balance_value"
+                style="@style/TextTitle"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="16dp"
+                android:layout_marginTop="4dp"
+                android:layout_marginEnd="16dp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/total_balance_label"
+                tools:text="10,679.25 USD" />
+
+            <androidx.appcompat.widget.AppCompatButton
+                android:id="@+id/send_button"
+                style="@style/PrimaryButton"
+                android:layout_width="wrap_content"
+                android:layout_height="42dp"
+                android:layout_marginTop="16dp"
+                android:layout_marginEnd="8dp"
+                android:layout_marginBottom="24dp"
+                android:drawableLeft="@drawable/ic_arrow_red_10dp"
+                android:drawableTint="@color/white"
+                android:drawablePadding="8dp"
+                android:paddingHorizontal="32dp"
+                android:text="@string/coins_send"
+                android:textSize="14dp"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toStartOf="@+id/receive_button"
+                app:layout_constraintHorizontal_chainStyle="packed"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/total_balance_value">
+
+                <aapt:attr name="android:background">
+
+                    <selector>
```

Comment:
Why is it embedded? Wouldn't it make sense to have it in a separate file in `drawable`?

Reply:
it is used only here. We don't have any other buttons with this style. For now having everything in one place makes adjuting and editing easier.

Reply:
![image](https://user-images.githubusercontent.com/246473/192782824-a7dc92ae-bc92-429e-8137-68673d7f0dcb.png)